pallets/identity/src/lib.rs

```diff
@@ -75,13 +75,12 @@
 //! - `add_authorization` - Adds an authorization.
 //! - `remove_authorization` - Removes an authorization.
 //! - `add_secondary_keys_with_authorization` - Adds secondary keys to target identity `id`.
-//! - `revoke_offchain_authorization` - Revokes the `auth` off-chain authorization of `signer`.
```

Comment:
"//! - `join_identity_as_identity` - Join an identity as a secondary identity." also needs to be removed from the comment above here